__tests__/app.test.js

```diff
@@ -49,6 +49,61 @@ describe("GET /api", () => {
 describe("GET /api/articles", () => {
   test("200 should respond with an article array or article objects with the correct properties", () => {
     return request(app)
+
+    .get("/api")
+    .expect(200)
+    .then(({body}) => {
+      expect(body).toEqual(apiEndpoints)
+    })
+  })
+})
+
+describe('GET /api/articles/:article_id/comments', () => {
+  test('200 accepts an article_id and responds with an array of comments for that given id ', () => {
+    return request(app)
+    .get("/api/articles/1/comments")
+    .expect(200)
+    .then(({body}) => {
+      const {comments} = body;
+      expect(comments).toHaveLength(11); 
+      expect(comments).toBeSorted({key: 'created_at', descending: true});
+      comments.forEach((comment) => {
+        expect(comment).toHaveProperty("comment_id", expect.any(Number));
+        expect(comment).toHaveProperty("votes", expect.any(Number));
+        expect(comment).toHaveProperty("created_at", expect.any(String));
+        expect(comment).toHaveProperty("author", expect.any(String));
+        expect(comment).toHaveProperty("body", expect.any(String));
+        expect(comment).toHaveProperty("article_id", expect.any(Number));
+      })
```

Comment:
it's worth noting that you can definitely be more specific about what article_id value comes back, these comments will all be for the same article